rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.stats/13_fields.yml

```diff
@@ -107,23 +84,6 @@ setup:
   - gt:       { _all.total.completion.size_in_bytes: 0 }
   - is_false:   _all.total.completion.fields
 
----
-"Fields - blank - global_ordinals":
-  - skip:
-      version: " - 8.7.99"
-      reason: "global_ordinals introduced in 8.8.0"
-
-  - do:
-      indices.stats: {}
-
-  - match: { _shards.failed: 0}
-  - gt:       { _all.total.fielddata.memory_size_in_bytes: 0 }
-  - is_false:   _all.total.fielddata.fields
-  - gte:       { _all.total.fielddata.global_ordinals.build_time_in_millis: 0 }
-  - is_false:   _all.total.fielddata.global_ordinals.fields
-  - gt:       { _all.total.completion.size_in_bytes: 0 }
```

Comment:
These are dropped below, just verifying that it's intended.

Reply:
I forgot to mention this. Yes, this is intended, there was duplication between tests.